content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -15,62 +15,114 @@ export class AccessCheckerForWearables {
     private readonly collectionsL2SubgraphUrl: string,
     private readonly blocksL1SubgraphUrl: string,
     private readonly blocksL2SubgraphUrl: string,
+    private readonly thirdPartySubgraphUrl: string,
     private readonly LOGGER: log4js.Logger
   ) {}
 
   public async checkAccess({ pointers, ...accessParams }: WearablesAccessParams): Promise<string[]> {
-    const errors: string[] = []
+    if (pointers.length !== 1) return [`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`]
 
-    if (pointers.length !== 1) {
-      errors.push(`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`)
+    const pointer: Pointer = pointers[0].toLowerCase()
+    const parsed = await this.parseUrnNoFail(pointer)
+    if (!parsed)
+      return [
+        `Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointer})`
+      ]
+
+    const { network, type } = parsed
+
+    const isL1Network = AccessCheckerForWearables.L1_NETWORKS.includes(network)
+    const isL2Network = AccessCheckerForWearables.L2_NETWORKS.includes(network)
+    if (!(isL1Network || isL2Network)) return [`Found an unknown network on the urn '${network}'`]
+
+    const isThirdparty = type === 'blockchain-collection-third-party'
+
+    if (isThirdparty) {
+      const existsItem = await this.checkItemExistence(
+        pointer,
+        this.thirdPartySubgraphUrl,
+        this.blocksL2SubgraphUrl,
+        accessParams
+      )
+      if (!existsItem) return [`The third-party item ${pointer} does not exist.`]
+      // @todo check if the third-party item is owne by the user
```

Comment:
Please validate with dApps team, but I believe that the below validation needs to be done whether or not it's a third-party wearable, the only difference is the subgraph URL.